extensions/redcore.xml

```diff
@@ -70,6 +70,25 @@
         </submenu>
     </administration>
 
+    <files folder="components/com_redcore/site">
+        <folder>controllers</folder>
+        <folder>helpers</folder>
+        <folder>layouts</folder>
+        <folder>models</folder>
+        <folder>tables</folder>
+        <folder>views</folder>
+        <filename>controller.php</filename>
+        <filename>redcore.php</filename>
+    </files>
+    
+    <languages folder="components/com_redcore/site/language">
```

Comment:
no, we kicked out language installation. Just put language folder in the files list above.